apps/api/src/app/subscribers-v2/e2e/get-subscriber-preferences.e2e.ts

```diff
@@ -0,0 +1,133 @@
+import { expect } from 'chai';
+import { randomBytes } from 'crypto';
+import { UserSession } from '@novu/testing';
+import { ChannelTypeEnum } from '@novu/shared';
+import { NotificationTemplateEntity } from '@novu/dal';
+import {
+  UpdateSubscriberGlobalPreferencesRequestDto,
+  UpdateSubscriberPreferenceRequestDto,
+  SubscriberResponseDto,
+} from '@novu/api/models/components';
+
+const v2Prefix = '/v2';
+let session: UserSession;
+
+describe('Get Subscriber Preferences - /subscribers/:subscriberId/preferences (GET) #novu-v2', () => {
```

Comment:
These should be updated and use the new API SDK and Novu client. Check the file `apps/api/src/app/subscribers-v2/subscribers.controller.e2e.ts`